variables.tf

```
@@ -49,3 +49,15 @@ variable "sns_email_subscription" {
   type        = string
   default     = ""
 }
+
+variable "cold_storage_after" {
+  description = "Number of days after which the backup is moved to cold storage"
+  type        = number
+  default     = null
+}
+
+variable "delete_after" {
+  description = "Number of days after which the backup is deleted"
+  type        = number
+  default     = null
+}
```

```suggestion
  default     = 100
}

```
Use the previous value as the default to minimize unexpected changes. (And end the file with a newline.)